src/server/hook_func.c

```diff
@@ -153,11 +153,11 @@ extern	char server_host[PBS_MAXHOSTNAME+1];
 /* Local Private Functions */
 
 /* Global Data items */
+int	do_sync_mom_hookfiles = 1;
```

Comment:
I had to look in to the usage of these variables to understand them. Hence asked. I couldn't figure when they will be true or false.